policyengine_canada/variables/gov/provinces/sk/tax/income/credits/sk_senior_supplementary_credit/sk_senior_supplementary_credit.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_canada.model_api import *
+
+
+class sk_senior_supplementary_credit(Variable):
+    value_type = float
+    unit = CAD
+    entity = Person
+    label = "Sasktachewan senior supplementary tax credit"
+    definition_period = YEAR
+    defined_for = ProvinceCode.SK
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.sk.tax.income.credits.sk_senior_supplementary
+        age = person("age", period)
+        eligibility = where(age >= p.age_threshold, 1, 0)
```

Comment:
make a separate sk_senior_supplementary_credit_eligible.py file for the eligibility condition it would only have 
        p = parameters(
            period
        ).gov.provinces.sk.tax.income.credits.sk_senior_supplementary
        age = person("age", period)
        return age >= p.age_threshold